src/Index.js

```diff
@@ -46,7 +46,7 @@ Index.prototype.addObject = function(content, objectID, callback) {
     hostType: 'write',
     callback: callback
   });
-};
+}, deprecatedMessage('index.addObject()', 'index.addObjects()'));
```

Comment:
Ah, didn't think of that, sounds like a good idea, but in the new version, would we ask for people to do: 

```js
index.addObjects([object]);

// or

index.addObjects(object);
```

I assumed you meant the second method, but I think I prefer the first code and consistency-wise